server/Program.cs

```diff
@@ -42,7 +42,8 @@ void ConfigureServices(WebApplicationBuilder builder)
   }
   builder.Services.AddDatabaseDeveloperPageExceptionFilter();
 
-  builder.Services.AddIdentity<ApplicationUser, IdentityRole>(options => options.SignIn.RequireConfirmedAccount = true)
```

Comment:
Let's stick to `AddIdentity`. I explored the two options yesterday and the main differences are the lack of pages in one and the lack of roles in the other. `AddDefaultIdentity` features a set of pages already configured out of the box, a lot of which we don't even need like registration or account management. It's better to go with `AddIdentity` for role support and implement our own admin portal login page.

Reply:
Alright will change it back and push